.github/CODEOWNERS

```diff
@@ -315,7 +315,7 @@
 /core/platform/ @seanlip
 /core/templates/dev/head/App*.ts @ankita240796
 /core/templates/dev/head/app.constants.ts @ankita240796
-/core/templates/dev/head/pages/interaction-specs.contants.ts @ankita240796
+/core/templates/dev/head/pages/interaction-specs.constants.ts @ankita240796
```

Comment:
Change to @vojtechjelinek in a future PR since @Jamesjay4199 is working on a refactor later I think.

Reply:
Done!